tests/conformance/state/state_test.go

```diff
@@ -17,10 +17,10 @@ type ValueType struct {
 }
 
 const (
-	maxInitDurationInMs   = 10
-	maxSetDurationInMs    = 10
-	maxGetDurationInMs    = 10
-	maxDeleteDurationInMs = 10
+	maxInitDurationInMs   = 1000
```

Comment:
These should change to 1000 * time.Millisecond

Reply:
I think each component should be able to pass in its own set configuration as the minimum. That we can refactor in later.